tests/ssg-mode.test.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @jest-environment node
```

Comment:
@ebisbe The magic line is this one
This tells jest that in this file stuff like `window` and `document` doesn't exist :slightly_smiling_face: 
So now we should check whatever we want to check

Currently I just copied the normal tests over and changed them a bit, but I'm not sure if this is sufficient